### DISCUSSION TITLE
czy lista filtrow jest aktualna?

### DISCUSSION THREAD

**Discussion by ghost** [MajkiIT/polish-ads-filter#21703] [General]
czy ta lista jest nadal aktualna, czy trzeba cos zmienic?
![filtry](https://user-images.githubusercontent.com/104568362/233559017-be975a8e-dea7-4787-81bf-2cbc2ef408fd.jpg)

**Comment by xxcriticxx** [comment]
@Lukjan39 co za dużo to nie zdrowo

**Reply by ghost** [reply]
czyli ok, lub cos zmienic?

**Reply by AroKol78** [reply]
**według mojej wiedzy** lista `KAD` i `CERT` (masz dwa razy listy `CERT`)są tym samym, jedna z nich jest zbędna, różnice:
- `KAD` zawiera już listę `CERT` z tą różnicą, że dodano autorskie filtry np. od sklepów oszustów, oraz jest sprzątana z nieaktywnych domen - według mnie "the best option"  👍 
- `CERT` też jest dobra, tylko, że nie jest sprzątana z zaszłości historycznych i dlatego jest większa od `KAD`

tu są proponowane listy https://majkiit.github.io/polish-ads-filter/
(dorzucił bym jeszcze `Dandelion Sprout's Anti-Malware List`)

**Reply by ghost** [reply]
aha, dobra
